src/task.js

```diff
@@ -0,0 +1,29 @@
+const fs = require('fs');
+const touch = require('touch');
+
+function registerTask(on, config) {
+  on('task', {
+    mkdir(newDir) {
+      return new Promise(function(resolve, reject) {
+        fs.mkdir(newDir, resolve);
```

Comment:
Perhaps I need to simulate the `mkdir -p` here, though in testing so far, i haven't needed it.